### PR TITLE
feat: Optional codemod for migrating enums

### PR DESCRIPTION
**Pull request by rivka-ungar**
https://monday.monday.com/boards/3532714909/views/80492480/pulses/7338744903

### CODE REVIEW

**Comment by YossiSaadi** [comment]
feat(icon)?

File: packages/codemod/bin/vibe-codemod.ts [hunk 1]
```diff
@@ -61,12 +61,20 @@ async function runWizard() {
       choices: [{ name: "jsx", checked: true }, { name: "tsx", checked: true }, { name: "js" }, { name: "ts" }],
       default: argv.extensions || ["jsx", "tsx"],
       when: !argv.extensions
+    },
+    {
+      type: "confirm",
+      name: "runOptional",
+      message: "Would you like to run the optional enum migrations as well?",
+      default: false,
+      when: true
```
Comment (talkor) [inline packages/codemod/bin/vibe-codemod.ts:70]: I think it should be a different kind of migration rather as a part of v3

**Review by talkor — COMMENTED** [review]
WDYT?

**Review by talkor — APPROVED** [review]
Amazing!! 💯 👑

File: packages/codemod/bin/vibe-codemod.ts [hunk 2]
```diff
@@ -112,6 +113,18 @@ async function main() {
     }
   }
 
+  const majorVibeVersion = getPackageMajorVersion("@vibe/core");
```
[the remark this answers is not in the document]
Comment (rivka-ungar) [reply packages/codemod/bin/vibe-codemod.ts]: Changed

**Review by talkor — APPROVED** [review]

**Pull request merged**